### PR TITLE
Change limits in SPM and off-nom-roll validators to fix violations

### PR DESCRIPTION
**Pull request by taldcroft**
## Description

This PR opens up a couple of validation limits to remove spurious violations.

For the sun position monitor, the original tolerance of 300 seconds was a little too tight and a few cases of up to 360 seconds occurred. This is related to the difference between predicted eclipse exit times and what the OBC actually did on orbit. This increases the limit to 400 sec.

For off-nominal roll, the error due to sampling the maneuver attitude at only 5-minute bins is a bit larger than was originally assumed. Increasing the tolerance during NMAN from 10 deg to 12 deg removes violations from a 200 day sample.

## Interface impacts

None.

### Unit tests

- [x] Mac (done at the first commit before fixing the notebook name)
This requires the file `kadi/commands/test/conftest.py` from [e99d43e](https://github.com/sot/kadi/commit/e99d43e8ff1371cd95ebff24e98cddbd524b2d37). This file was used in testing and inadvertently left out of the commits.
```
(ska3-perf) ➜  kadi git:(fix-spm-off-nom-roll-violations) git rev-parse HEAD      
467afb24b99214851778deb97789d5b508032034
(ska3-perf) ➜  kadi git:(fix-spm-off-nom-roll-violations) pytest
==================================================== test session starts =====================================================
platform darwin -- Python 3.10.8, pytest-7.2.1, pluggy-1.0.0
rootdir: /Users/aldcroft/git, configfile: pytest.ini
plugins: timeout-2.1.0, anyio-3.6.2
collected 219 items                                                                                                          

kadi/commands/tests/test_commands.py ...........................................................................       [ 34%]
kadi/commands/tests/test_states.py ......................x.............................................x.............. [ 72%]
.........                                                                                                              [ 76%]
kadi/commands/tests/test_validate.py ....................                                                              [ 85%]
kadi/tests/test_events.py ..........                                                                                   [ 89%]
kadi/tests/test_occweb.py ......................                                                                       [100%]

========================================= 217 passed, 2 xfailed in 94.04s (0:01:34) ==========================================
```

Independent check of unit tests by Jean with that conftest.py and with a ska3-masters environment and dev/local versions of chandra_aca, ska_sun, and chandra_maneuver in PYTHONPATH. 
- [x] Linux

### Functional tests

See the included notebook.

### CODE REVIEW

File: validate/pr305-func-test.ipynb [hunk 1]
```diff
@@ -0,0 +1,2731 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Functional testing for PR-302\n",
```
Comment (jeanconn) [inline validate/pr305-func-test.ipynb:7]: Should probably be PR-305 here but not sure if it is worth the update.

**Comment by jeanconn** [comment]
Technically I think this needs a rebase to get that conftext.py and pass tests.

**Comment by taldcroft** [comment]
> Technically I think this needs a rebase to get that conftest.py and pass tests.

`conftest.py` is in master so when this is merged the `master` branch will pass. I've updated the documentation to reflect that the testing results depend on having that file from `master` in the correct place.

**Comment by jeanconn** [comment]
That's fine.  My process is always just to check out the PR and run tests; so for this I'll copy the conftest and run tests if that's the only change that is missing for tests to pass.

**Pull request merged**